cmd/kubeadm/app/cmd/upgrade/apply.go

```diff
@@ -30,6 +31,7 @@ import (
 
 	kubeadmapi "k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm"
 	"k8s.io/kubernetes/cmd/kubeadm/app/cmd/options"
+	cmdutil "k8s.io/kubernetes/cmd/kubeadm/app/cmd/util"
```

Comment:
kubeadm now has two `util` packages, not sure whether we can merge them.

`"k8s.io/kubernetes/cmd/kubeadm/app/cmd/util"`
`"k8s.io/kubernetes/cmd/kubeadm/app/util"`

Reply:
Cmd / backend util separation is usually good to have. I have not looked at the current state but we can keep them for now.

Reply:
thanks for the reply!